granulate_utils/metrics/sampler.py

```diff
@@ -269,7 +261,12 @@ def _init_collectors(self):
         """
         if self._cluster_mode == SPARK_YARN_MODE:
             self._spark_samplers.append(YarnCollector(self._master_address, self._logger))
-        elif self._cluster_mode == SPARK_STANDALONE_MODE or self._cluster_mode == SPARK_MESOS_MODE:
+
+        # In Standalone and Mesos we'd use applications metrics
+        if self._cluster_mode in (SPARK_STANDALONE_MODE, SPARK_MESOS_MODE):
+            self._applications_metrics = True
+
+        if self._applications_metrics is True:
```

Comment:
Do not use `is True` / `is False`. It is not customary.

Reply:
493f35963d9fdb4056e587f57176efbebc31606c